activation.cpp

```diff
@@ -106,10 +143,31 @@ auto Activation::activation(Activations value) -> Activations
         using UpdateAccessKey = phosphor::software::image::UpdateAccessKey;
         UpdateAccessKey updateAccessKey(manifestPath);
 
+        using VersionClass = phosphor::software::manager::Version;
```

Comment:
We only want to check uak on oob updates so we want to check if the staging dir is empty, we can check it right after the ifdef above in line 139.